app/controllers/staff/activities_controller.rb

```diff
@@ -4,7 +4,9 @@ class Staff::ActivitiesController < Staff::BaseController
   include Secured
 
   def index
-    @activities = policy_scope(Activity).includes(:parent)
+    @activities = policy_scope(Activity.project_activities)
+    authorize @activities
+    @activity_presenters = @activities.includes([:organisation, :parent]).map { |activity| ActivityPresenter.new(activity) }
```

Comment:
Since the `includes` relates the to the ActiveRecord query rather than the presenter, how about rewriting as:

```ruby
@activities = policy_scope(Activity.project_activities).includes([:organisation, :parent])
authorize @activities
@activity_presenters = @activities.map { |activity| ActivityPresenter.new(activity) }
```

Reply:
We talked about this one and why the includes live where it is.